web/css/source/module/homepage/_brands.less

```diff
@@ -0,0 +1,8 @@
+.brands h3{
+    position: absolute;
+    margin-left:25%;
+    background:white;
+}
+.brands hr{
+    margin-top: 3rem;
+}
```

Comment:
All general styles (for both mobile and desktop) should be wrapped with a `& when (@media-common = true) {}` construction

Reply:
Also use nesting, like:
```
.brands {
  h3 {
      position: absolute;
      margin-left:25%;
      background:white;
  }
  hr {
      margin-top: 3rem;
  }
}
```